src/structs/channels.rs

```diff
@@ -543,6 +549,17 @@ bitflags! {
     const FAILED_TO_MENTION_SOME_ROLES_IN_THREAD = 1 << 8;
     /// This message will not trigger push and desktop notifications
     const SUPPRESS_NOTIFICATIONS = 1 << 12;
+    /// This message is a voice message
+    const IS_VOICE_MESSAGE = 1 << 12;
```

Comment:
voice messages are `1 << 13`

Reply:
Oops, I copy pasted the last line but only changed the name :p